setup.py

```diff
@@ -36,8 +36,6 @@
 ]
 extras = {
     "grpc": ["grpcio >= 1.33.2, < 2.0dev", "grpcio-status >= 1.33.2, < 2.0dev"],
-    "grpcgcp": "grpcio-gcp >= 0.2.2, < 1.0dev",
```

Comment:
Removing an extra might be considered a breaking change. You might want to add these back but with an empty dependency list.

Reply:
Or possibly not empty but as an alias for the gcpio extra?

Reply:
As discussed over GVC, a missing extra might just raise a warning, not an error.